en/3_Open_Source_Software_Use.md

```diff
@@ -1,4 +1,4 @@
-[« Previous Page : Open Standards](2_Open_Standards.md) | [Table of Contents](../README.md#table-of-contents) | [Next Page: Open Source Software Contribution »](4_Open_Source_Software_Contribution.md)
+[« Previous Page : Open Standards](2_Open_Standards.md) | [Table of Contents](../README.md#table-of-contents) | [Next Page: Open Source Software Contribution »](4_Open_Source_Software_Contribution)
```

Comment:
Droping the `.md` makes this invalid. Not sure why the build is passing with this

Reply:
Thanks @nschonni ! Found a whole lot of other dead links! Really weird that the script didn't pick those up...